### ISSUE TITLE
Request: a release that works with Go 1.19 or Go 1.20

### ISSUE THREAD

**Issue by bsiegert** [ipfs/ipget#132]
The two Go versions supported by upstream are 1.19 and 1.20. The last ipget version, 0.9.1, does not build with either.

```
$ go version
go version go1.20.2 netbsd/amd64
$ go install -v github.com/ipfs/ipget@latest
[...]
# github.com/lucas-clemente/quic-go/internal/qtls    
go/pkg/mod/github.com/lucas-clemente/quic-go@v0.27.1/internal/qtls/go119.go:6:13: cannot use "The version of quic-go you're using can't be built on Go 1.19 yet. For more details, please see https://github.com/lucas-clemente/quic-go/wiki/quic-go-and-Go-versions." (untyped string constant "The version of quic-go you're using can't be built on Go 1.19 yet. F...) as int value in variable declaration 
```

Installing the latest checkout from git is not a viable workaround for packaging systems like pkgsrc. pkgsrc packages only released versions by policy.

A simple thing that could be done would be to create a branch at `v0.9.1`, update quic-go and perhaps other modules, then publish that as `v0.9.1.1`. This would not require bringing the current main branch into a shape fit for releasing.

**Comment by gammazero** [comment]
This was fixed by #119. Update to ipget v0.9.2 to build with go1.19 or go1.20.

**Comment by bsiegert** [comment]
Thanks! https://pkgsrc.se/net/ipget is now updated.
